environment.yml

```diff
@@ -5,6 +5,8 @@ dependencies:
   - python >=3.9,<3.13
   # Don't forget to sync changes between environment.yml, environment-dev.yml, and pyproject.toml!
   # Main packages
+  - botocore == 1.34.106
+  - aiobotocore == 2.13.0
```

Comment:
Why exactly are these hard pinned? This needs to be resolved before merging.

Reply:
@Zeitsperre I included this pin because there's a difference between the environments.  This cause a problem in the tests for test_gis.py when they are run in github. How can I solve it?
![image](https://github.com/user-attachments/assets/0108386e-861a-4283-96a8-da69c953e1ae)